tests/newman/tests/admin-api.json

```diff
@@ -481,7 +503,8 @@
 									"",
 									"pm.test(\"Expect non-null methodology to have correct fields\", function () {",
 									"    for(var i = 0; i < respJson.length; i++) {",
-									"        if(respJson[i].methodology !== null) {",
+									"        console.log(\"method \", respJson[i].methodology);",
```

Comment:
Do you wanna keep this console.log() or remove?

Reply:
Removed